rtc/encryption.md

```diff
@@ -0,0 +1,70 @@
+
+> **Warning**: The key hint is stored by the server in plain text. Do not add secret or sensitive information to the key hint.
+
+The key hint can be a key thumbprint, ID, or other non-sensitive identifier that will help select the key, such as a timestamp. It is only recorded when the protocol requests a new key. Returning a key hint from requests for previously issued keys will have no effect.
+
+## RTC encryption details
+
+> **Note**: This section contains the technical details of encryption used to securely transmit document contents. It is provided for information purposes only; understanding these details is not required to integrate RTC.
+
+The RTC protocol does not use the [provided encryption key]({{site.baseurl}}/rtc/configuration#rtc_encryption_provider) to encrypt content directly. Using industry standard cryptography algorithms a unique session key for content encryption is derived from the provided key in a way that makes brute force decryption very difficult.
+
+> **Note**: {{site.companyname}} is in the process of changing RTC encryption to [AES-KW](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/wrapKey) which does not require an initialization vector or protocol salt. The description below is still correct for the current beta release.
+
+The RTC protocol encryption technique is as follows:
+* 256 bits of random data are generated as the salt for each session. As described above, each document ID used for collaboration may have multiple sessions. The salt is generated with the [Crypto.getRandomValues()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues) browser API and a `Uint8Array` of length 32 bytes.
+* The salt data and the provided encryption key are combined using the [PBKDF2](https://en.wikipedia.org/wiki/PBKDF2) key derivation function. Derivation is performed by the [SubtleCrypto.deriveKey()](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/deriveKey) browser API, using the following parameters:
+  * PBKDF2 [algorithm](https://developer.mozilla.org/en-US/docs/Web/API/Pbkdf2Params), with SHA-512 digest algorithm and 10000 iterations
+  * AES-GCM [derivedKeyAlgorithm](https://developer.mozilla.org/en-US/docs/Web/API/AesKeyGenParams) with length 256
+* The derived key is cached (in memory only) and used with [SubtleCrypto.encrypt()](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/encrypt) and [SubtleCrypto.decrypt()](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/decrypt) as necessary, specifying the [AES-GCM algorithm](https://developer.mozilla.org/en-US/docs/Web/API/AesGcmParams). For each encrypted message 96 bits of random data are used as the initialization vector, also generated with [Crypto.getRandomValues()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues) but this time using `Uint8Array` length 12. Each IV is transmitted alongside the encrypted message.
```

Comment:
```suggestion
* The derived key is cached (in memory only) and used with [SubtleCrypto.encrypt()](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/encrypt) and [SubtleCrypto.decrypt()](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/decrypt) as necessary, specifying the [AES-GCM algorithm](https://developer.mozilla.org/en-US/docs/Web/API/AesGcmParams). For each encrypted message, 96 bits of random data are used as the initialization vector, also generated with [Crypto.getRandomValues()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues) but this time using `Uint8Array` with a length of 12. Each IV is transmitted alongside the encrypted message.
```

Reply:
> Each IV is transmitted alongside the encrypted message.

What's `IV`?

Reply:
Initialization Vector. I guess I can repeat that rather than using the acronym.